scipy/io/mmio.py

```diff
@@ -284,17 +287,41 @@ def _get_symmetry(a):
         issymm = 1
         isskew = 1
         isherm = a.dtype.char in 'FD'
-        for j in range(n):
-            for i in range(j+1,n):
-                aij,aji = a[i][j],a[j][i]
-                if issymm and aij != aji:
-                    issymm = 0
-                if isskew and aij != -aji:
-                    isskew = 0
-                if isherm and aij != conj(aji):
-                    isherm = 0
-                if not (issymm or isskew or isherm):
-                    break
+        
+        if isinstance(a,spmatrix):
```

Comment:
Another way to check for sparse matrix without using `isinstance` ?    Maybe presence of `nnz` attribute?